core/src/main/scala/com/lightbend/paradox/markdown/Directive.scala

```diff
@@ -381,6 +381,7 @@ case class FiddleDirective(page: Page, variables: Map[String, String])
           |  // $FiddleEnd
           |}
           |""".stripMargin, "UTF-8")
+        .replace("+", "%20") // due to: https://stackoverflow.com/questions/4737841/urlencoder-not-able-to-translate-space-character
```

Comment:
I wonder if we could use some utilities from akka-http to build up the properly-formatted URL without having to use `URLEncoder` here, which is not really meant for this purpose

Reply:
If you agree on having akka-http dependency to this sbt plugin it could be worth exploring this possibility of course

Reply:
Ah sorry I thought for a moment we already had that dependency - let's not add it.

Not super nice but not sure introducing a dependency just for this is warranted, so 👍